glusterd2/events/liveness.go

```diff
@@ -57,23 +56,34 @@ func (l *livenessWatcher) Watch() {
 					"peer.id": peerID,
 				}
 
-				Broadcast(New(evName, data, false))
+				err = Broadcast(New(evName, data, false))
 			}
-		case <-l.stopCh:
+		case errc := <-l.stopCh:
+			errc <- err
+			l.stopCh = nil
 			return
 		}
 	}
 }
 
+func (l *livenessWatcher) Stop() error {
+	if l.stopCh == nil {
+		return fmt.Errorf("livness watcher has not been started")
```

Comment:
TRIVIAL : Can we use errors.New() instead of fmt.Errorf() ?. As we have been using it throughout the gd2 code, it would look uniform.